sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/CodeGenerator.scala

```diff
@@ -910,12 +910,14 @@ class CodegenContext {
     val blocks = new ArrayBuffer[String]()
     val blockBuilder = new StringBuilder()
     var length = 0
+    val splitThreshold =
+      SQLConf.get.getConfString("spark.testing.codegen.splitThreshold", "1024").toInt
```

Comment:
It seems that this is not a good way, cc @dongjoon-hyun

Reply:
+1 for @wangyum 's comment.
@yucai . In this case, we can not call this `[TEST]` or `Refactor`.

Reply:
How about adding ```val splitThreshold =
      SQLConf.get.getConfString("spark.testing.codegen.splitThreshold", "1024").toInt``` to our `To run this benchmark`?

Reply:
@wangyum Thanks for the suggestion! You prefer to modifying `CodeGenerator.scala` each time run this benchmark, right? I feel it could be kind of tricky to let user modify codes and if the `CodeGenerator.scala` changes in the future, it is hard to update the document here. @dongjoon-hyun @gengliangwang any suggestion?

Reply:
In this case, we need advice from the right person. :)

Reply:
Personally I don't think this is a good solution:
1. The configuration contains "testing", which is super wired as it can be used in production.
2. We should start a new discuss about whether to make it configurable. The reason should not be making the benchmark easier.